docs/quickstart.md

```diff
@@ -11,6 +11,19 @@ The AEA framework can be used on `Windows`, `Ubuntu/Debian` and `MacOS`.
 
 You need <a href="https://www.python.org/downloads/" target="_blank">Python 3.6</a> or higher as well as <a href="https://golang.org/dl/" target="_blank">Go 1.14.2</a> or higher installed.
 
+
+### Optional: using the Docker image.
+
+We also provide a Docker image with all the needed dependencies.
+
+To pull:
+```bash
+docker pull marcofavorito/aea-deploy:latest
+```
+
```

Comment:
To run the image
​
- Linux and MacOs
  ```bash
  docker run -it -v $(pwd):/dev marcofavorito/aea-ubuntu:latest 
  ```
- Windows
  ```bash
  docker run -it -v %cd%:/dev marcofavorito/aea-ubuntu:latest 
  ```
Once succefully logged into the docker container, 
you can follow the rest of the guide the same way as if not using docker.
​